playground/src/editor.tsx

```diff
@@ -206,6 +208,7 @@ export function Editor(props: {
       sources: [["code", code]],
       languageFeatures: [],
       profile: profile,
+      projectType: "qsharp",
```

Comment:
Is this a breaking change for QCOM? (i.e. will they need to pass this extra param now also). If not (i.e it's optional/unnecessary), why not just leave this file untouched?